package.json

```diff
@@ -1,38 +1,61 @@
 {
-  "name": "cesium-webpack-example",
+  "name": "cesiumjs-webpack-example",
   "version": "1.0.0",
   "description": "The minimal recomended setup for an app using Cesium with Webpack.",
-  "main": "index.js",
+  "homepage": "https://cesium.com/platform/cesiumjs/",
+  "license": "Apache-2.0",
+  "author": {
+    "name": "Cesium GS, Inc.",
+    "url": "https://cesium.com"
+  },
+  "contributors": [
```

Comment:
@srothst1 We don't need this section in package.json.

Reply:
Sounds good - I will remove the "contributors" section.